app/forms/document_type_upload_form.rb

```diff
@@ -19,7 +18,7 @@ def save
   def instantiate_document
     document_file_upload = attributes_for(:intake)[:document]
     if document_file_upload.blank?
```

Comment:
After removing the file_type_allowed validator, I feel like this may only push an error if the document_type is blank... it seems like we potentially would want to do both.